README.md

```diff
@@ -1,239 +1,233 @@
 
-As with BigInt, there's not a huge amount of precedent from other programming languages about a literal suffix. Newer languages in the .NET ecosystem tend to use `m`, but outside of that family, few languages use a suffix to indicate decimal literals. So there isn't a lot of precedent to go on.
+## Syntax and semantics
 
-Many people have raised the idea that we use `d` as the suffix for decimal. It is the first letter! However, thinking about this further, it would not extend to hexadecimal literals, which may include `d`.
+With Decimal we do not envision a new literal syntax. One could consider one, such as `123.456_789m` is a Decimal value ([#7](https://github.com/tc39/proposal-decimal/issues/7)), but we are choosing not to add new syntax in light of feedback we have received from JS engine implementors as this proposal has been discussed in multiple TC39 plenary meetings.
 
-Although this proposal does not include hexadecimal syntax for decimals, it would be ideal to work with a syntax that could be extended to them, or even to "explain" the decimal syntax in terms of a generalized [extended numeric literals](https://github.com/tc39/proposal-extended-numeric-literals) proposal (which would necessarily support hexadecimal literals, to explain BigInt).
+### Data model
 
-### Would fractions/rationals meet these use cases?
+Decimal represents a mathematical, “normalized” ([#26](https://github.com/tc39/proposal-decimal/issues/26)) base 10 decimal. IEEE 754 Decimal128 is the underlying set of values (though not all Decimal128 values are available to the JS programmer, since we work, again, only with normalized values, whereas the official IEEE 754 Decimal128 works with unnormalized values). For example, `2`, as a Decimal, is exactly the same value as `2.00m` ([#11](https://github.com/tc39/proposal-decimal/issues/11)) If preserving magnitude/precision through trailing zeroes is required, it needs to be represented separately from the Decimal. ([#9](https://github.com/tc39/proposal-decimal/issues/9))
 
-Fractions would be an interesting thing to pursue in TC39, and are in many ways complementary to Decimal. Many languages in the Lisp tradition include fractions of arbitrary-size integers as a basic data type, alongside IEEE-754 64-bit binary floating point numbers; Ruby and Python also include fractions in their standard library.
+### Operator semantics
 
-We see fractions as complementary to Decimal because of a mismatch when it comes to two of the core operations on Decimals:
-- Rounding to a certain base-10 precision, with a rounding mode
-- Conversion to a localized, human-readable string
++ Addition, multiplication, subtraction, division, and remainder are defined.
++ Bitwise operators are not supported, as they don’t logically make sense on the Decimal domain ([#20](https://github.com/tc39/proposal-decimal/issues/20))
++ No trigonometric functions (and hence no hyperbolic trigonometric functions, either)
```

Comment:
Above you say that supporting them is a goal

Reply:
Oops, that's a mistake. Fixed now. Thanks!